internal/pkg/template/templates/environment/cf.yml

```diff
@@ -397,8 +420,7 @@ Outputs:
     Export:
       Name: !Sub ${AWS::StackName}-SubDomain
   EnabledFeatures:
-    # We don't need to include Aliases because updating it always results in the CustomDomain action to update.
-    Value: !Sub '${ALBWorkloads},${EFSWorkloads},${NATWorkloads}'
+    Value: !Sub '${ALBWorkloads},${EFSWorkloads},${NATWorkloads},${Aliases}'
```

Comment:
Why did we include `Aliases`? I'm a little worried that this will cause the environment updates to hit limits too frequently.

Reply:
> I'm a little worried that this will cause the environment updates to hit limits too frequently.

it would be the same behavior as is 🤔 because atm `CustomDomainAction` get constantly updated whenever they specify new aliases. Alternatively, we could update the env controller so that it won't pass any alias to the env controller if env has cert imported.

Reply:
Let's do it I think it would be a better option

Reply:
> it would be the same behavior as is 🤔 

Oh I meant the Output value limit sorry (which I think is probably 4096 bytes).

> Let's do it I think it would be a better option

That sounds good to me too - but I didn't understand why you updated this value originally 💭 not that it matters if you're going to revert it 😝

Reply:
> That sounds good to me too - but I didn't understand why you updated this value originally 💭

I think we might still want to add it. The reason is because: it is always guaranteed rn that whenever a new alias is added to the env stack, at least one resource (`CustomDomainAction`) will get updated so that our env controller won't get "no stack update" error. However, if an environment imports any certificates, `CustomDomainAction` won't be created and when service is switching the alias it is using, env controller will return an no-update error without the code change we are talking about. That's the initiative why I added this.

Alternatively, like I mentioned above, we could modify the input for env controller so that i won't add new aliases to the env stack if it imports any certificates. However, it would be a problem that the env stack `aliases` param won't have a full set of all the aliases which potentially could be a problem for our operational commands that grab alias info from there.

Or we could let env controller ignore the `no-update` error when it is updating the env stack, but that could also lead to unexpected consequences because we are ignoring a type of error.

> Oh I meant the Output value limit sorry (which I think is probably 4096 bytes).

That is true, but since the parameter length limit is also 4096 bytes, most likely even if we go with another route they'll face the same issue unless we use SSM or other places to store their aliases rather than using the CFN parameter.

Reply:
Discussed offline! we'll fetch the alias from the manifest and can remove this change 🥳